realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -284,6 +288,16 @@ protected final void copy(ColumnInfo rawSrc, ColumnInfo rawDst) {
     }
 
     @Override
+    public RealmInteger realmGet$columnCounter() {
+        return new ManagedRealmInteger(0);
+    }
+
+    @Override
+    public void realmSet$columnCounter(RealmInteger value) {
```

Comment:
This feels wrong like @zaki50 said. But maybe you plan to change it in a following PR?